### PR TITLE
Fix panic on cacheACL.Fields["meter"] type assertion

### LINKED ISSUE #80: panic on cacheACL type assertion in example
os:
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

ovn-nbctl (Open vSwitch) 2.9.5
DB Schema 5.10.0

ovn-sbctl (Open vSwitch) 2.9.5
DB Schema 1.15.0

#### how to reproduce:

go run examples/main.go

#### error message
```bash
{b7c93ac6-d263-47dc-bd9b-d1e7c10da7d5 test  map[] [12:34:56:78:90 10.10.10.1] []   map[]}
panic: interface conversion: interface {} is nil, not libovsdb.OvsSet

goroutine 1 [running]:
github.com/ebay/go-ovn.(*ovndb).rowToACL(0xc0000b2120, 0xc00001c4e0, 0x24, 0x0)
        /home/max/GolandProjects/go-ovn/acl.go:280 +0xa31
github.com/ebay/go-ovn.(*ovndb).aclListImp(0xc0000b2120, 0x74cc9c, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/max/GolandProjects/go-ovn/acl.go:322 +0x36c
github.com/ebay/go-ovn.(*ovndb).ACLList(0xc0000b2120, 0x74cc9c, 0x3, 0x1, 0x0, 0x0, 0x51, 0x74cdf7)
        /home/max/GolandProjects/go-ovn/client.go:426 +0x3f
main.main()
        /home/max/GolandProjects/go-ovn/examples/main.go:70 +0x803

### PR DESCRIPTION
**Pull request by fyuan1316**
Fix #80

### CODE REVIEW

File: acl.go [hunk 1]
```diff
@@ -277,8 +277,10 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
 	case string:
 		severity = cacheACL.Fields["severity"].(string)
 	case libovsdb.OvsSet:
-		for _, a := range cacheACL.Fields["meter"].(libovsdb.OvsSet).GoSet {
-			severity = a.(string)
+		if meters, ok := cacheACL.Fields["meter"]; ok {
```
Comment (noah8713) [inline acl.go]: Thanks for catching this. @fyuan1316 Can you also do this check for severity and other meter usage in rowToACL function? Just move this if meters, ok := cacheACL.Fields["meter"]; ok  before swith case https://github.com/eBay/go-ovn/blob/master/acl.go#L265

File: acl.go [hunk 2]
```diff
@@ -277,8 +277,10 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
...
+			for _, a := range meters.(libovsdb.OvsSet).GoSet {
+				severity = a.(string)
+			}
 		}
```
Comment (hzhou8) [inline acl.go]: Thanks for the PR. However, this fix doesn't make sense. The problem is apparently a copy/paste bug. The fix should replace "meter" with "severity".

**Comment by fyuan1316** [comment]
Fixed copy/paste errors. Any other suggestions?

**Comment by hzhou8** [comment]
Since it is a one line fix for typo, I merged it directly. Thanks for the fix.

**Pull request merged**